HOW_WE_USE_GITHUB.md

```diff
@@ -265,6 +267,75 @@ Community support can be found elsewhere, though, and we encourage you to explor
 
 In order to not have to manually type or copy/paste the above repeatedly, note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option][docs-saved-reply].
 
+## Development Processes
+
+### How should we approach feature development?
+
+For new features, begin by creating documentation for the new feature within
+the project so that you can get early input on the design. Documentation
+includes design mockups, API reference docs, recording the planned approach
+in the feature issue, and creating CLI argument demos without the
+functionality of the feature.
+
+### What is our change process?
+
+For larger features, break down the work into smaller, manageable issues
+that are added to the backlog. As long as a feature remains on the roadmap
+or backlog, do not create long-lived feature branches that span multiple
+pull requests. Instead, you should integrate small slices of an overall
+feature directly into the main branch to avoid complex integration challenges.
+
+### Should we make unrelated changes at the same time?
+
+When making changes, try to follow the Campsite Rule to leave things better
+than when you found them. You should enhance the code you encounter, even if
+primary goal is unrelated. This could involve refactoring small sections,
+improving readability, or fixing minor bugs.
+
+## Code Review and Merging
+
+### What are the review requirements?
+
+#### Standard Review
+
+Most code changes require one reviewer from someone on the maintainer team for
+the repository. Instead of waiting for someone on the team to review it,
+directly requesting a review from the person you previously identified to work
+with is preferred to optimize teamwork. If you paired with them during
+development, continuous review counts as this requirement.
+
+#### Second Review
+
+Required only when the code author or the first reviewer feels like it is
+necessary to get another set of eyes on a proposed change. In this case, they
+add someone specific through GitHub's Request Review feature with a comment on
+what they want the person to look for.
+
+### What are the code review best practices?
+
+If you are conducting a review, adhere to these best practices:
+
+- Provide comprehensive feedback in the first review to minimize review rounds
```

Comment:
```suggestion
- Provide comprehensive feedback in the first review to minimize review rounds
- For core reviews, do not simply "comment" on the review; either submit a review as
   "approve" or "request changes" to clearly communicate to the author what is needed
```

This is a practice that seems like would help during our reviews. Happy to discuss here if anyone disagrees though.

Reply:
I really like that we are discussing this and can make this practice more consistent on when we do certain types of review selections.

The other problem with Request Changes is someone else can't approve of the code with out clearing the review. Clearing a review requires entered justification which makes it feel like you are breaking the rules. This slows down cadence unless the first reviewer is ready to re-review quickly.

I'm thinking request changes should be reserved for things that would be harmful to the code base if merged as is. I added another suggestion to capture this.